Makefile.in

```diff
@@ -207,10 +207,7 @@ crypto/test/cipher_driver$(EXE): crypto/test/cipher_driver.c test/getopt_s.c
 crypto/test/kernel_driver$(EXE): crypto/test/kernel_driver.c test/getopt_s.c
 	$(COMPILE) $(LDFLAGS) -o $@ $^ $(LIBS) $(SRTPLIB)
 
-crypto/test/rand_gen$(EXE): crypto/test/rand_gen.c test/getopt_s.c
```

Comment:
These are removed because it is dead code right? Files are nowhere to be found at least.

Reply:
yip, as far as I understood.